src/types/Room.ts

```diff
@@ -1,8 +1,11 @@
-import { Color } from './Color.ts';
+import Coordinates from './Coordinates.ts';
 
+/**
+ * Room data
+ */
 export default interface Room {
 	id: number;
 	name: string;
-	points: number[][];
-	visualizationData: Color;
+	locationArrays: Coordinates[];
```

Comment:
Name of "locationArrays" is very ambiguous: Locations of what does this array represent?

Reply:
Renamed to roomBounds